### PR TITLE
Fix 0011738: Setting a confirm_action for an ability generates a Y/N …

### PR DESCRIPTION
**Pull request by ghost**
…prompt every time the ability menu is opened

The confirm prompt was still being generated when
_check_ability_possible was called in "quiet" mode, though it shouldn't
have been.

### CODE REVIEW

**Comment by ebering** [comment]
Oh! And how would you like to appear in CREDITS.txt?

**Comment by ebering** [comment]
Sorry, it seems I spoke too soon. Your patch has whitespace trailing at line ends, you can use the `util/checkwhite` utility in the source directory to clean these up.

File: crawl-ref/source/ability.cc [hunk 1]
```diff
@@ -1420,21 +1420,24 @@ static bool _check_ability_possible(const ability_def& abil, bool quiet = false)
         return false;
     }
 
-    vector<text_pattern> &actions = Options.confirm_action;
-    if (!actions.empty())
+    if (!quiet) 
     {
```
Comment (NormalPerson7) [inline crawl-ref/source/ability.cc]: Or you can manually fix the trailing whitespace - it's just a single space after this `if`.

**Comment by ebering** [comment]
I'm adding you to CREDITS.txt as GenericPseudonym for now, if you'd like this changed, send us a PR!

**Pull request closed without merging**